Clarify Clock terminology: replace "Clock tick" with "Clock activation" (#2107)
modelica/fmi-standard: docs/2_2_common_mechanisms.adoc

Clocks are <<discrete>> <<fmu-variables,variables>> of type `fmi3Clock`.
They are restricted to being scalar variables.

Clock tick::

> A super-dense time instant at which a Clock is active (i.e., the Clock's activation state is `fmi3ClockActive`). A Clock tick denotes the precise time point — including its super-dense ordering — during which clocked variables and the associated model partition may be evaluated.

Author: Replaced "tick" with "activation"

Clock activation::
The action of changing a Clock's state to `fmi3ClockActive`.

The variable's attribute <<intervalVariability>> declares the type of Clock, see overview in <<table-overview-clocks>>.
After <<table-overview-clocks>> Clock types are explained in more detail.

[[inputClock, `input Clock`]]
Input Clock::
+
is a variable of type Clock with <<causality>> = <<input>>.

> In Model Exchange and Co-Simulation, when input Clocks are activated, the importer will activate the input Clocks using <<fmi3SetClock>>.

Reviewer: ```suggestion
In Model Exchange and Co-Simulation, input Clocks are activated by the using <<fmi3SetClock>>.
```

Reviewer 2: ```suggestion
In Model Exchange and Co-Simulation, input Clocks are activated by the importer using <<fmi3SetClock>>.
```

Clocks are <<discrete>> <<fmu-variables,variables>> of type `fmi3Clock`.
They are restricted to being scalar variables.

Clock activation::
The action of changing a Clock's state to `fmi3ClockActive`.
A Clock activation denotes the precise super-dense time instant — including its super-dense ordering — during which clocked variables and the associated model partition may be evaluated.

The variable's attribute <<intervalVariability>> declares the type of Clock, see overview in <<table-overview-clocks>>.
After <<table-overview-clocks>> Clock types are explained in more detail.

[[inputClock, `input Clock`]]
Input Clock::
+
is a variable of type Clock with <<causality>> = <<input>>.

> In Model Exchange and Co-Simulation, input Clocks are activated by the the importer using <<fmi3SetClock>>.

Reviewer: ```suggestion
In Model Exchange and Co-Simulation, input Clocks are activated by the importer using <<fmi3SetClock>>.
```

|[[changing-aperiodic-clock]]<<fmi3SetClock>> in <<EventMode>>,
<<fmi3ActivateModelPartition>> in <<ClockActivationMode>>,
<<fmi3GetInterval>> in <<InitializationMode>>,

> <<fmi3GetInterval>> in both <<EventMode>> and <<ClockActivationMode>> if and only if this Clock was activated

Reviewer 3: If I remember correctly, in PR https://github.com/modelica/fmi-standard/pull/2050 , we said getInterval may be  called in any event mode and may affect non-active clock intervals too.

Author: Replaced "tick" with "activation"

[cols="1,5"]
|===
|latexmath:[t_0]

> |The time instant of the first Clock tick.

Author: Replaced "tick" with "activation"
